oasis/libs/rules.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+import re
+
+OPERATORS = ["==", "!=", "<=", ">=", "<", ">"]
+
+FEATURE_RULE = r'^features\[(\w+)\]\s*(==|!=|<=|>=|=|<|>)\s*(\d+|\.)'
```

Comment:
Why not simply `std > 100`, removing `features`?

Reply:
the rules is not only `features rule` in the future，I just add `features` to diff them